client/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Signin from './component/signin/Signin';
+import Signin from './component/signin/index';
```

Comment:
It imports by default from the index so you don't need to write it. just import from the signing folder